### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by JevanChandra**
Please help review my code 

@AdriantoHartono
@a-rosa
@Riston_gitting_gud
@MikeLeongZhiFai

### CODE REVIEW

File: cp1404practicals/prac_09/band.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/band.py:1]: Great code👌

File: cp1404practicals/prac_09/car.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/car.py:1]: Nice code👌

File: cp1404practicals/prac_09/silver_service_taxi.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/silver_service_taxi.py:1]: Great code👍

File: cp1404practicals/prac_09/silver_service_taxi_test.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/silver_service_taxi_test.py:1]: Nice code👍

File: cp1404practicals/prac_09/taxi.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/taxi.py:1]: Great code, the code has been written neatly🙏

File: cp1404practicals/prac_09/taxi_simulator.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/taxi_simulator.py:1]: The code has been written neatly in each function👌

File: cp1404practicals/prac_09/taxi_test.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/taxi_test.py:1]: Great code

File: cp1404practicals/prac_09/unreliable_car.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/unreliable_car.py:1]: Nice code👌

File: cp1404practicals/prac_09/unreliable_car_test.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_09/unreliable_car_test.py:1]: Great code

File: cp1404practicals/prac_09/unreliable_car.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_09/unreliable_car.py:1]: Code is nicely written!

File: cp1404practicals/prac_09/unreliable_car_test.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_09/unreliable_car_test.py:1]: Nice code!

File: cp1404practicals/prac_09/band.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        self.musicians.append(f"{member.name} ({member.instruments})")
+
+    def play(self):
+        message = []
+        for musician in self.musicians_dictionary:
+            if not self.musicians_dictionary[musician]:
+                message.append(f"{musician} needs an instrument!")
+            else:
+                message.append(f"{musician} is playing: {self.musicians_dictionary[musician][0]}")
+        return "\n".join(message)
```
Comment (a-rosa) [inline cp1404practicals/prac_09/band.py:21]: Missing docstring

File: cp1404practicals/prac_09/silver_service_taxi.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    price_per_km = Taxi.price_per_km
+    flagfall = 4.50
+
+    def __init__(self, name, fuel, fanciness):
+        super().__init__(name, fuel)
+        self.price_per_km = self.price_per_km * fanciness
```
Comment (a-rosa) [inline cp1404practicals/prac_09/silver_service_taxi.py:10]: can be shorten to self.price_per.km *= fanciness

File: cp1404practicals/prac_09/taxi_simulator.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+def get_valid_taxi(current_taxi, chosen_taxi, taxis):
+    if current_taxi < 0 or current_taxi > len(taxis) - 1:
+        print("Invalid taxi choice")
+        chosen_taxi = None
+    else:
+        chosen_taxi = current_taxi
+    return chosen_taxi
+
+
+main()
```
Comment (a-rosa) [inline cp1404practicals/prac_09/taxi_simulator.py:54]: Missing docstring on the functions

**Pull request merged**